dexcom/egv.go

```diff
@@ -171,33 +182,39 @@ func NewEGV(unit *string) *EGV {
 }
 
 func (e *EGV) Parse(parser structure.ObjectParser) {
+	e.ID = parser.String("recordId")
 	e.SystemTime = TimeFromRaw(parser.ForgivingTime("systemTime", TimeFormat))
 	e.DisplayTime = TimeFromRaw(parser.ForgivingTime("displayTime", TimeFormat))
 	e.Value = parser.Float64("value")
```

Comment:
Interesting, the old code never picked up the `Unit` field. Should be added here?

Reply:
Looks like the unit tests never covered that, so it was probably never discovered.

Reply:
So maybe it wasn't specified when the work was done, although it is shown in dexcom's `v2`? I will add it in

Reply:
Ahhh ok figured it out.

v2 at the top level specified the `unit` and `rateUnit`. This has now been pushed down into each individual reading in v3. Thanks for the question though and I will clean that up